tests/unit/util/rbtree_test.c

```diff
@@ -292,11 +292,12 @@ test_case_pre(struct mtf_test_info *ti)
 int
 test_case_post(struct mtf_test_info *ti)
 {
-    struct rb_node *node;
+    struct rb_node *node, *next;
 
-    for (node = rb_first(&tree); node; node = rb_next(node)) {
+    for (node = rb_first(&tree); node; node = next) {
```

Comment:
Use after free, so get the next node prior to freeing the encapsulating container.

Reply:
Prob better to use a for-each-safe loop here...  But realistically, no real need to tear down the tree, just use a for-each loop to walk the tree and free each entry, then discard the tree...

Reply:
Is there some macro you are referring to?

Reply:
The linux rbtree abomination doesn't have RB_FOREACH(), so just use rbtree_postorder_for_each_entry_safe() which iterates sequentially from last to first in a safe manner (i.e., you can delete each node you visit as you go...)

Reply:
If you wanna drop this fix onto my PR go ahead...